wangle/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@
 
 cmake_minimum_required(VERSION 2.8)
 
-set(CMAKE_CXX_FLAGS "-std=c++0x -fPIC")
+set(CMAKE_CXX_FLAGS "-std=c++14 -fPIC")
```

Comment:
does  -std=c++1y work for you? That would keep from moving the minimum gcc to 4.9